src/darjeeling/evaluator.py

```diff
@@ -130,7 +130,7 @@ def _filter_redundant_tests(self,
         drop = set()  # type: Set[Test]
         for test in tests:
             test_line_coverage = line_coverage_by_test[test.name]
-            if not any(line in test_line_coverage for line in lines_changed):
+            if len(lines_changed)>0 and not any(line in test_line_coverage for line in lines_changed):
```

Comment:
It would be great if we could remove this clause and instead introduce a new `if` statement before the `for` statement. Maybe something like:
```
# if no lines are changed, retain all tests (fixes issue #128)
if not lines_changed:
  return (tests, set())
```

Reply:
yeah, with that set of changes, the intention is completely clear. looks good to me.

Reply:
there we go. Thanks @ChrisTimperley!